command/agent/config.go

```diff
@@ -1329,7 +1333,8 @@ func DefaultConfig() *Config {
 				LimitResults:  100,
 				MinTermLength: 2,
 			},
-			JobMaxSourceSize: pointer.Of("1M"),
+			JobMaxSourceSize:   pointer.Of("1M"),
+			JobTrackedVersions: pointer.Of(6),
```

Comment:
We've left `structs.JobTrackedVersions` untouched, so it would probably make more sense for this to be `pointer.Of(structs.JobTrackedVersions)`, right? But also, we should also rename `structs.JobTrackedVersions` to indicate it's the now the default (and update its docstring).

Reply:
Done